rpm_spec/gui-agent.spec.in

```diff
@@ -159,6 +169,58 @@ echo autospawn=no >> /etc/pulse/client.conf
 rm -rf $RPM_BUILD_ROOT
 rm -f %{name}-%{version}
 
+%if %{with selinux}
+%package selinux
+Summary: SELinux policy for the Qubes GUI Agent
+License: GPLv2+
+
+# The following is based on %%selinux_requires, which has the following license:
```

Comment:
Same comment as in https://github.com/QubesOS/qubes-linux-utils/pull/80

Reply:
And same answer :laughing: